docs/Actions.md

```diff
@@ -2204,3 +2204,18 @@ rsync(
   destination: "/tmp/local_file.txt"
 )
 ```
+
+### `load_plugin`
+
+If you want to keep your action implementation separate from the `fastlane` main code base, you can provide it as a remote plugin.
+
+More information in [Extensions.md](https://github.com/fastlane/fastlane/blob/master/docs/Extensions.md).
+
+To import a remote plugin use
+
+```ruby
+lane :beta do
+  load_plugin(url: "https://github.com/fastlane/plugin_example")
+  remote_plugin # that's the action that is stored in this git repo
```

Comment:
1. can you have multiple actions in one repo ?
2. why load_plugin instead of load_action[s] ?
3. and why not have an Actionfile and move all initializations there ?

Something like
``` 
actions (url: "https://github.com/fastlane/plugin_example") # implies all from master
actions (url: "https://github.com/fastlane/plugin_example2", "1.0",  # 2 specific from a particular tag
  [ 
     'remote_plugin1'
     'remote_plugin2'
   ]
```

is there any point of having the flexibility of having each lane specify its action sources and plugins to load ?